Neos.Media/Classes/Domain/Model/Thumbnail.php

```diff
@@ -31,6 +31,7 @@
 class Thumbnail implements ImageInterface
 {
     use DimensionsTrait;
+    use QualityTrait;
```

Comment:
I notice this is only applied to thumbnails, in theory it should be applied to all images regardless if they are scaled or not right?